ICSharpCode.Decompiler/TypeSystem/ApplyAttributeTypeVisitor.cs

```diff
@@ -220,8 +220,8 @@ Nullability GetNullability()
 
 		void ExpectDummyNullabilityForGenericValueType()
 		{
-			var n = GetNullability();
```

Comment:
What's the point of this? You cannot just comment out code that does not work in your case!